functions/db/deliveryRequests/onCreate.f.js

```diff
@@ -4,6 +4,8 @@ const enums = require('../../Enums.js');
 const utils = require('../../Utils.js');
 
 const nt = enums.NotificationType;
+// initalizes firebase so that you can access firebase references
+const admin = require('firebase-admin');
```

Comment:
Is this needed? I don't see anywhere where this admin const is used but I might be misunderstanding its purpose